index.html

```diff
@@ -9863,7 +9884,7 @@ <h4 id="Beast Shape - Medium Animal">Beast Shape - Medium Animal</h4>
                </td>
                <td>Pack Hunter
                </td>
-               <td>Do additional Light damage for every ally adjacent to the enemy you are
+               <td>Do an additional 1 damage for every ally adjacent to the enemy you are
```

Comment:
I'm still not ok with this as a passive, it needs to be a limited use ability. We can't have consistent sources of additional damage.